### PR TITLE
SN-30|SN-31/Active Storage

### PR DESCRIPTION
**Pull request by skripchenkoVeta**
**_Description:_**
_SN-30_ - task: https://trello.com/c/8Imj1zfD
_SN-31_ - task: https://trello.com/c/NyaNq6uI

### CODE REVIEW

**Comment by skripchenkoVeta** [comment]
There are no tests and no seeds due to the fact that the polymorphic association requires an object

**Review by jtethras — CHANGES_REQUESTED** [review]
I don't see changes to `schema.rb`
Define `picture` factory (https://stackoverflow.com/questions/55141549/attaching-activestorage-files-in-factorybot)
Add some specs (i.e. `Picture` model expects to have one attached picture)
Add some pictures to `seeds`

**Review by Slavahaim — APPROVED** [review]

File: app/models/picture.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Picture < ApplicationRecord
+  belongs_to :picturable, polymorphic: true
+  has_one_attached :picture
```
Comment (ramantechart) [inline app/models/picture.rb]: please add validation that Picture record contains file

File: spec/models/picture_spec.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Picture, type: :model do
+  let(:invalid) { FactoryBot.build :picture, :invalid }
+  let(:empty) { FactoryBot.build :picture, :empty }
+
+  context 'with valid attributes' do
+    it { expect(invalid).to_not be_valid }
```
Comment (ramantechart) [inline spec/models/picture_spec.rb]: :D

File: .rubocop.yml [hunk 3]
```diff
@@ -16,6 +16,7 @@ AllCops:
     - 'Rakefile'
     - 'Gemfile.lock'
     - 'config.ru'
+    - 'db/migrate/20220120101655_create_active_storage_tables.active_storage.rb'
```
Comment (PFDeve) [inline .rubocop.yml:19]: we can just add `'db/**/*'` or `'db/migrate/*'`

Comment (ramantechart) [reply .rubocop.yml:19]: @PFDeve @skripchenkoVeta I suggest don't any of the approaches.
Instead please user `.rubocop-todo.yml`

Comment (ramantechart) [reply .rubocop.yml:19]: Lets fix it in the next PR

**Review by ramantechart — APPROVED** [review]

**Pull request merged**